openassessment/templates/legacy/response/oa_response.html

```diff
@@ -281,6 +281,7 @@ <h5 class="message__title">{% trans "We could not delete files" %}</h5>
                                 </div>
                                 <input type="file" class="submission__answer__upload file--upload" id="submission_answer_upload_{{ xblock_id }}" {% if allow_multiple_files %}multiple=""{% endif %} accept="{{white_listed_file_types|join:", "}}">
                                 <button type="submit" class="file__upload action action--upload">
+                                  <span class="icon fa fa-spinner fa-spin is--hidden" aria-hidden="true"></span>
```

Comment:
This is working great, but the `is--hidden` style isn't working so well, so the spinner is causing the button text to be off center:

![image](https://github.com/user-attachments/assets/49245d06-8da5-439e-84e7-1cdea9dddf4d)

Could you update [`%state-hidden`](https://github.com/openedx/edx-ora2/blob/3418d7b74e0a20d6ced041dc0e0175b300f32934/openassessment/xblock/static/sass/xb/utilities/_extends.scss#L124) to make `display: none !important;`?